src/services/API/Event.js

```diff
@@ -10,4 +14,12 @@ export default {
         return response.data;
       });
   },
+  /**
+   *
+   * @param {Number} eventId - The ID of the event that needs to be patched
+   * @param {Object} eventData - The event will be patched with this data
+   */
+  updateEvent(eventId, eventData) {
+    return apiClient().patch(eventsEndpoint + eventId, eventData);
```

Comment:
we are sending the entire event details object, no? then, put will be more appropriate no?